### PR TITLE
Fixes and reenables rubocop Performance/StringReplacement.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Fixes generated by `bundle exec rubocop --only Performance/StringReplacement --auto-correct`.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
FYI: http://ruby-doc.org/core-1.9.3/String.html#method-i-tr, http://stackoverflow.com/questions/26749065/what-is-the-difference-between-tr-and-gsub.

File: dashboard/app/controllers/admin_search_controller.rb [hunk 1]
```diff
@@ -93,8 +93,8 @@ def search_for_teachers
       # Remove newlines from the full_address field, replacing them with spaces.
       @teachers.each do |teacher|
         if teacher[3].present?
-          teacher[3].gsub!("\r", ' ')
-          teacher[3].gsub!("\n", ' ')
+          teacher[3].tr!("\r", ' ')
+          teacher[3].tr!("\n", ' ')
```
Comment (joshlory) [inline dashboard/app/controllers/admin_search_controller.rb:97]: Nit: could be combined into one statement as `teacher[3].tr!("\r\n", ' ')`.

**Comment by joshlory** [comment]
LGTM.  I assume the CircleCI build failure is unrelated?

**Comment by ashercodeorg** [comment]
Presumably, though I'm rerunning to get a green run to be safe.

**Comment by ashercodeorg** [comment]
CircleCI is an unrelated timeout.

**Pull request merged**